projects/plugins/jetpack/extensions/blocks/cookie-consent/view.js

```diff
@@ -10,7 +10,7 @@ domReady( function () {
 	const expireTimeDate = new Date( Date.now() + expireDays * millisInDay );
 	const cookieName = 'eucookielaw';
 
-	const button = container.querySelector( '.wp-block-button a' );
+	const button = container.querySelector( '.wp-block-button' );
```

Comment:
Since we're changing the saved block structure, I think we'll need to add a deprecation handler:
https://developer.wordpress.org/block-editor/reference-guides/block-api/block-deprecation/

You can find some examples in this repo by searching for `deprecated` in `projets/plugins/jetpack/extensions`.

Adding this will avoid block validation errors when trying to open the site editor on a site that previously added a consent block.

Reply:
Working on it.

Reply:
@jeherve thanks for all feedback. You have angel blood 🥇

Reply:
This is not relevant now since we don't alter the saved content or change the structure.
We just changed the selector to target the button link